src/spritekit.cs

```diff
@@ -149,8 +149,10 @@ partial interface SKNode : NSSecureCoding, NSCopying {
 		[return: NullAllowed]
 		SKNode Create (string filename);
 
+#if MONOMAC || WATCH
```

Comment:
I don't understand that change. I believe this is a breaking change. Also I'm confused about a SpriteKit change in this UIKit PR.

Reply:
No, it is not. In iOS and TV we have the IUIFocusItem interface that already contains the [Frame property](https://github.com/xamarin/xamarin-macios/pull/4253/files#diff-1ca9e8e46629c0f8806cee2c8be4aad3R16730). That means that in those platforms, it can be removed. 

On WatchOS and MacOS we do not have that interface, therefore the property does not get generated and you have to add it manually.

Reply:
@VincentDondain once the build is done, take a look at the api diff and the Extrospection results both confirm my statement :)

Reply:
Ah got it, didn't realize it was in `IUIFocusItem` 👍